### PR TITLE
Add nearbyClaims flag to VisualizationEvent

### PR DESCRIPTION
**Pull request by FrankHeijden**

### CODE REVIEW

File: src/main/java/me/ryanhamshire/GriefPrevention/events/VisualizationEvent.java [hunk 1]
```diff
@@ -63,6 +78,15 @@ public boolean showSubdivides()
         return showSubdivides;
     }
 
+    /**
+     * Check if event was called through shift-inspecting with the inspection tool.
+     * @return True if shift-inspecting
+     */
+    public boolean isNearbyClaims()
```
Comment (RoboMWM) [inline src/main/java/me/ryanhamshire/GriefPrevention/events/VisualizationEvent.java]: hasMultipleClaims
isVisualizingNearbyClaims

idk, isNearbyClaims is a little too small to be self-descriptive but eh, I can't really think of a better one...

**Review by RoboMWM — COMMENTED** [review]
Hmm, I see... thought he called the event in the Visualization class. Idk if I like the name but eh.

File: src/main/java/me/ryanhamshire/GriefPrevention/events/VisualizationEvent.java (hunk 1 above)
Comment (FrankHeijden) [reply src/main/java/me/ryanhamshire/GriefPrevention/events/VisualizationEvent.java]: I think your second option is also nice. However if we'd look from the context with the class name:
`VisualizationEvent#isNearbyClaims` and `VisualizationEvent#isVisualizingNearbyClaims`

I think they are both fine

Comment (RoboMWM) [reply src/main/java/me/ryanhamshire/GriefPrevention/events/VisualizationEvent.java]: idk, the former fits better in that context.

**Comment by FrankHeijden** [comment]
My unix-like system really doesn't like the DOS-eol's lol

**Comment by RoboMWM** [comment]
What's up with this file? My default git settings autotranslate line endings so that's probably why I don't observe issues on Windows, but then again I haven't worked with that file in a while.

**Comment by FrankHeijden** [comment]
> What's up with this file? My default git settings autotranslate line endings so that's probably why I don't observe issues on Windows, but then again I haven't worked with that file in a while.

Really interesting indeed, was able to fix it for the Visualization class, but trying the same settings on this file doesn't seem to work...

**Comment by FrankHeijden** [comment]
Idk, am out of options here

**Comment by RoboMWM** [comment]
git is configured to autoconvert line endings for me so that class shows up as CRLF on Windows, and I guess git autoconverts it back to LF on commit... or, it's CRLF and is remaining as such. Hmm.

**Comment by FrankHeijden** [comment]
Finally, lol

**Review by RoboMWM — APPROVED** [review]

**Pull request merged**